tensorflow_serving/g3doc/serving_config.md

```diff
@@ -140,3 +140,35 @@ the model config on the fly, to assign a label to it
 (can be achieved using
 [HandleReloadConfigRequest](https://github.com/tensorflow/serving/blob/master/tensorflow_serving/apis/model_service.proto#L22)
 RPC endpoint).
+
+# How to Configure for Monitoring
+
+Creating a monitoring config file containing a
+[MonitoringConfig](https://github.com/tensorflow/serving/blob/1.13.0/tensorflow_serving/config/monitoring_config.proto#L17)
+and set its path to --monitoring_config_file. It currently provides a monitoring
+exporter for [Prometheus](https://prometheus.io/), and whose configs are
+described
+[PrometheusConfig](https://github.com/tensorflow/serving/blob/1.13.0/tensorflow_serving/config/monitoring_config.proto#L7)
+protocol buffer. Here's an example:
+
+```proto
+prometheus_config {
+  enable: true
+  path: '/metrics'
```

Comment:
i would skip `path` and mention that  monitoring endpoint is at `/monitoring/prometheus/metrics`. this will encourage people to use the default path. customizing path should be a rare thing than a common one. and not mentioning that in an example will be nice.